client/src/ui/molecules/document-survey/surveys.ts

```diff
@@ -95,29 +97,15 @@ export const SURVEYS: Survey[] = [
     end: Infinity,
   },
   {
-    key: SurveyKey.HOUSE_SURVEY_2025,
-    bucket: SurveyBucket.HOUSE_SURVEY_2025,
+    key: SurveyKey.JS_SURVEY_1_2025,
+    bucket: SurveyBucket.JS_SURVEY_1_2025,
     show: (doc: Pick<Doc, "mdn_url">) =>
-      /^\/[^/]+\/docs\/(Web|Learn_web_development)($|\/.*$)|/i.test(
-        doc.mdn_url
-      ),
-    src: "https://survey.alchemer.com/s3/8145183/MDN-short-survey",
+      /^\/[^/]+\/docs\/Web($|\/.*$)/i.test(doc.mdn_url),
+    src: "https://survey.alchemer.com/s3/8234511/MDN-javascript-proposals-2025",
     teaser:
-      "We'd love to hear more about your role and the company you work for",
-    question: "Please help us by answering a few questions.",
-    ...survey_duration(SurveyBucket.HOUSE_SURVEY_2025),
-    ...survey_rates(SurveyKey.HOUSE_SURVEY_2025),
-  },
-  {
-    key: SurveyKey.WEBDX_EDITING_2024,
-    bucket: SurveyBucket.WEBDX_EDITING_2024,
-    show: (doc: Pick<Doc, "mdn_url">) =>
-      /^\/[^/]+\/docs\/Web\/API($|\/.*$)/i.test(doc.mdn_url),
-    src: "https://survey.alchemer.com/s3/8045877/MDN-text-edit-survey",
-    teaser:
-      "Let us know your thoughts about the following text editing, formatting, selection, and clipboard APIs",
-    question: "Please share your feedback about the following features.",
-    ...survey_duration(SurveyBucket.WEBDX_EDITING_2024),
-    ...survey_rates(SurveyKey.WEBDX_EDITING_2024),
+      'We’d love to hear your thoughts on the next set of proposals for the JavaScript language. You can find a <a href="https://docs.google.com/document/d/1DMPXS4Po5Nd-l0mIEYj78H1S_R5FrB7CkHbluhqQPVo/" target="_blank">description of the proposals here</a>.',
```

Comment:
Nit: Should we shorten this?

```suggestion
      'We’d love to hear your thoughts on the <a href="https://docs.google.com/document/d/1DMPXS4Po5Nd-l0mIEYj78H1S_R5FrB7CkHbluhqQPVo/" target="_blank">next set of proposals</a> for the JavaScript language.',
```

PS: Either way I think this could be confusing. Could the proposal description not be linked from the survey?

Reply:
I think we keep the more explicit sentence (even if it is longer) - and I do agree this _might_ be confusing - but at the same time I'm interested to see if we get a drop/increase in responses because we're able to explian possibly unknown features 👍